cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -43,39 +43,61 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
       );
     });
 
-    // Have the test toggle between if mediaPlayer is enabled on live or not
-    let imageIndex = 1;
-    if (appToggles.mediaPlayer.enabled) {
-      imageIndex = 2;
-    }
-
     if (serviceHasCaption(service)) {
```

Comment:
Is this check still needed given the tests on L74-?

Reply:
Nope, it not needed hence why its omitted.